### PR TITLE
Improvements to Import Directory

### PR DESCRIPTION
**Pull request by ajd4096**
Some improvements to the Import Directory UI, and a post-processing tag to handle split layouts like "Movies/A/A Great Movie"

### CODE REVIEW

File: static/js/library/import/directory.js [hunk 1]
```diff
@@ -106,6 +119,8 @@ function connect(event, elem){
                     var title_column = movie["title"];
                     if (movie["tmdbid"]){
                         title_column = `<a href="https://www.themoviedb.org/movie/${movie["tmdbid"]}">${title_column}</a>`
+                    } else {
+                        title_column = `<a href="https://www.themoviedb.org/search/movie?query=${title_column}" target="_blank">Search ${title_column}</a>`
```
Comment (barbequesauce) [inline static/js/library/import/directory.js:123]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6020720)

**Comment by barbequesauce** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 3
           

Complexity increasing per file
==============================
- core/ajax.py  9
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6020720&bid=19735113)

File: static/js/library/import/directory.js [hunk 2]
```diff
@@ -78,7 +91,7 @@ function connect(event, elem){
     var last_response_len = false;
     $.ajax(url_base + "/ajax/scan_library_directory", {
         method: "POST",
-        data: {"directory": directory, "minsize": minsize, "recursive": recursive},
+        data: {"directory": directory, "minsize": minsize, "recursive": recursive, "skipduplicatedirs": skipduplicatedirs, "maxresults": maxresults},
```
Comment (barbequesauce) [inline static/js/library/import/directory.js:94]: Codacy found an issue: [Expected property shorthand.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6020720)

Comment (barbequesauce) [inline static/js/library/import/directory.js:94]: Codacy found an issue: ['directory' is not defined.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6020720)

**Comment by scambra** [comment]
I have changed maxresults to allow empty value, which will display all found files, and set minimum to 1 (I think empty to get all is better than 0 to get all for users), and left default to empty, as it was previous behaviour.

**Pull request merged**